src/modules/package.rs

```diff
@@ -247,6 +247,15 @@ fn get_shard_version(context: &Context, config: &PackageConfig) -> Option<String
     format_version(raw_version, config.version_format)
 }
 
+fn get_daml_project_version(context: &Context, config: &PackageConfig) -> Option<String> {
+    let file_contents = context.read_file_from_pwd("daml.yaml")?;
+
+    let daml_yaml = yaml_rust::YamlLoader::load_from_str(&file_contents).ok()?;
+    let raw_version = daml_yaml.first()?["version"].as_str()?;
```

Comment:
Please use `.get("version")`.

Reply:
I believe the same applies here as https://github.com/starship/starship/pull/4004#discussion_r878699757, but happy to receive further feedback of that not being the case.